version.sbt

```diff
@@ -1 +0,0 @@
-version in ThisBuild := "0.13.8-SNAPSHOT"
```

Comment:
This changes the local value of `version` to be formatted like this 0.13.7+34-2d333018-SNAPSHOT. Is that an acceptable change? I think it's possible to get configure the version to not include the git describe information you prefer to keep the old behavior.

Reply:
imo i think it's ok.